src/main/java/org/mozilla/magnet/scanner/BaseScanner.java

```diff
@@ -1,8 +1,5 @@
-package org.mozilla.magnet.net.scanner;
```

Comment:
Leave the `net` package, or rename it to `network`.

If later on we need to do more native libraries we will add them to another namespace like `org.mozilla.magnet.notifications`